specification/search/data-plane/Azure.Search/preview/2020-06-30/searchservice.json

```diff
@@ -5795,7 +5795,250 @@
           "description": "A dictionary of indexer-specific configuration properties. Each name is the name of a specific property. Each value must be of a primitive type."
         }
       },
-      "description": "Represents parameters for indexer execution."
+      "description": "Represents parameters for indexer execution.",
```

Comment:
Actually line 5790, but GitHub won't let me comment on surrounding code 😣: add a `$ref` to this model from `configuration`.

Reply:
Oh sorry, I thought you wanted them completely decoupled by leaving the configuration's type as object.

Reply:
This will be a breaking change then, no?

Reply:
Fixed in commit https://github.com/Azure/azure-rest-api-specs/pull/10349/commits/8c52a2039638429b1ff8d53d13a39ba0d1bddd94